zeebe/scheduler/src/main/java/io/camunda/zeebe/scheduler/future/ActorFuture.java

```diff
@@ -126,6 +190,8 @@ default CompletableFuture<V> toCompletableFuture() {
    */
   <U> ActorFuture<U> andThen(Function<V, ActorFuture<U>> next, Executor executor);
```

Comment:
:thought_balloon: I guess this could have a default implementation that piggiebacks on the new `andThen` that can handle errors too.